src/compas_timber/assembly/assembly.py

```diff
@@ -20,13 +20,16 @@ class TimberAssembly(Assembly):
         A list of the keys of the beams included in this assembly.
     joint_keys :  list(int)
         A list of the keys of the joints included in this assembly.
+    topologies :  list(dict)
+        A list of JointTopology for assembly. dict is: {"detected_topo": detected_topo, "beam_a": beam_a, "beam_b":beam_b} See :class:`~compas_timber.connections.JointTopology`.
```

Comment:
to avoid circular references it'd be better if you store the beams' keys instead of the instances themselves.